pkg/trivyk8s/trivyk8s_test.go

```diff
@@ -345,3 +351,112 @@ func TestInitResources(t *testing.T) {
 		})
 	}
 }
+
+type kubectlAction func() error
+
+func TestListSpecificArtifacts(t *testing.T) {
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(10*time.Minute))
+	defer cancel()
+
+	k3sContainer, err := k3s.Run(ctx, "rancher/k3s:v1.27.1-k3s1")
+	require.NoError(t, err)
+	testcontainers.CleanupContainer(t, k3sContainer)
+
+	kubeConfigYaml, err := k3sContainer.GetKubeConfig(ctx)
+	require.NoError(t, err)
+
+	configPath := path.Join(t.TempDir(), "kubeconfig")
+	err = os.WriteFile(configPath, kubeConfigYaml, 0644)
+	require.NoError(t, err)
+
+	provider, err := testcontainers.ProviderDocker.GetProvider()
+	require.NoError(t, err)
+
+	images := []string{
+		"nginx:1.14.1",
+		"nginx:1.27.4",
+	}
+
+	for _, image := range images {
+		err = provider.PullImage(ctx, image)
+		require.NoError(t, err)
+	}
+
+	tests := []struct {
+		name              string
+		namespace         string
+		resources         []string
+		kinds             []string
+		action            kubectlAction
+		expectedArtifacts []*artifacts.Artifact
+	}{
+		{
+			"good way for pod",
+			"default",
+			[]string{filepath.Join("testdata", "single-pod.yaml")},
+			[]string{"pod"},
+			nil,
+			[]*artifacts.Artifact{
```

Comment:
can you add struct fields when initializing a test case? It helps with readability when not on an IDE.

Reply:
sure, It'll be better
done https://github.com/aquasecurity/trivy-kubernetes/pull/448/commits/639d6ff05668e16a388da6d98f68d5bb07ab0e68